src/database/seeds/testimonials.ts

```diff
@@ -17,4 +18,6 @@ export async function seed(): Promise<void> {
       created_at: new Date(),
```

Comment:
I think we can use` faker.date.past()` instead of` new Date()` , previously it's have been made me confused when getting returning data.

Reply:
it's better to approach the real case, because it uses a faker past, the timestamp is random
![image](https://user-images.githubusercontent.com/41193120/141724516-ba115f05-a722-4fff-9c2e-1a08ca026ab0.png)